### PR TITLE
*: add some unit tests and remove unused code

### PR DESCRIPTION
**Pull request by amyangfei**
### What problem does this PR solve? 
add some unit tests, mainly in `pkg` and `reparo/syncer module`

### What is changed and how it works?
add test cases to cover some not tracked code

### Check List 

Tests 

 - Unit test

### CODE REVIEW

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

**Comment by july2993** [comment]
/run-all-tests tidb=release-3.0 tikv=release-3.0 pd=release-3.0

File: pkg/util/util_test.go [hunk 1]
```diff
@@ -119,6 +119,11 @@ func (s *getAddrIPSuite) TestShouldRetIPV4(c *C) {
 		IP: net.ParseIP("192.168.1.2"),
 	}
 	c.Assert(getAddrDefaultIP(&addr), Equals, "192.168.1.2")
+
+	addr2 := net.IPAddr{
```
Comment (july2993) [inline pkg/util/util_test.go]: what's the propose of this added test, seems same as the "192.168.1.2" before

Comment (amyangfei) [reply pkg/util/util_test.go]: one is `net.IPNet` while new added is `net.IPAddr`

**Pull request merged**